com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1077,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- GWT Plugin debug launching -->
+   <extension
```

Comment:
As discussed, it would be good if this extension point - besides the DebugLaunch class contribution - also expects a text property, which describes what would be launched. I would expect the GWT-EclipsePlugin to use this text property as a label in the context menu - I guess ATM the label is hard-coded, which is not so ideal...